WordPress/Classes/ViewRelated/Activity/ActivityDetailViewController.swift

```diff
@@ -57,26 +57,18 @@ class ActivityDetailViewController: UIViewController {
     private func setupFonts() {
         nameLabel.font = UIFont.systemFont(ofSize: UIFont.preferredFont(forTextStyle: .footnote).pointSize,
                                            weight: .semibold)
-
-        if FeatureFlag.extractNotifications.enabled == false {
-            textLabel.font = UIFont.systemFont(ofSize: UIFont.preferredFont(forTextStyle: .body).pointSize,
-                                               weight: .semibold)
-        }
     }
 
     private func setupViews() {
         guard let activity = activity else {
             return
         }
 
-        let showFormattedText = FeatureFlag.extractNotifications.enabled
-        textLabel.isHidden = showFormattedText
-        textView.isHidden = !showFormattedText
+        textLabel.isHidden = true
```

Comment:
It appears that `textLabel` is always hidden. What do you think about removing it altogether?